apps/test/unit/code-studio/pd/workshop_dashboard/workshopFactory.js

```diff
@@ -33,14 +42,158 @@ Factory.define('csp summer workshop')
   .attr('scholarship_workshop?', true)
   .attr('location_name', 'physical');
 
+Factory.define('csp ayw workshop 1')
+  .extend('workshop')
+  .attr('course', 'CS Principles')
+  .attr('subject', 'Academic Year Workshop 1')
+  .attr('sessions', () => Factory.buildList('session', 5))
+  .attr('account_required_for_attendance?', true)
+  .attr('scholarship_workshop?', true)
+  .attr('location_name', 'physical');
+
+Factory.define(
+  'csp summer workshop starting within month of middleOfYearFakeToday'
```

Comment:
I think you can safely shorten this up a bit, since this is text that's going to get reused a lot:
```suggestion
  'csp summer workshop starting within a month'
```
and just use the more specific text when near the end of the year

Reply:
Sounds good!